scss/_patterns_navigation.scss

```diff
@@ -64,6 +54,16 @@ $navigation-hover-opacity: 0.58;
           left: 0;
           position: absolute;
           right: 0;
+          top: 0;
+
+          @media (min-width: $breakpoint-navigation-threshold) {
+            content: unset;
```

Comment:
I think in theory having `content: unset` would be enough to make the border not visible even if other properties are set.

Of course, to be sure all styles are reset on desktop your approach is better. That's one of the drawbacks of mobile first styles. If we would ever need to unset/override something more complicated we could probably consider other solution.